src/REAXFF/fix_qeq_reaxff.cpp

```diff
@@ -1158,7 +1158,7 @@ void FixQEqReaxFF::get_chi_field()
     for (int i = 0; i < nlocal; i++) {
       if (mask[i] & efgroupbit) {
         if (region && !region->match(x[i][0],x[i][1],x[i][2])) continue;
-        chi_field[i] = -efield->efield[i][3];
+        chi_field[i] = efield->efield[i][3];
```

Comment:
Does this need to be propagated to other styles like ACKS2 and KOKKOS?

Reply:
It seems that acks2/reaxff and the KOKKOS versions of qeq/reaxff and acks2/reaxff call `get_chi_field()` from `FixQEqReaxFF`, so I don't think any further changes are needed.